Sources/Fluent/QueryBuilder/QueryBuilder+Model.swift

```diff
@@ -29,15 +29,19 @@ extension QueryBuilder where Result: Model, Result.Database == Database {
     /// - returns: A `Future` containing the created `Model`.
     public func create(_ model: Result) -> Future<Result> {
         Database.queryActionApply(Database.queryActionCreate, to: &query)
-        var copy: Result
-        if Result.createdAtKey != nil || Result.updatedAtKey != nil {
-            // set timestamps
-            copy = model
+        var copy = model
+        
+        // set timestamps
+        if Result.createdAtKey != nil || Result.createdAtKey != nil {
```

Comment:
This looks like you meant something else for the part after the `||`

Reply:
Why? You mean `// set timestamps` comment? This is original comment..